cv32e40s/tests/programs/custom/clic/clic.c

```diff
@@ -3408,6 +3482,16 @@ __attribute__((interrupt("machine"))) void u_sw_irq_handler(void) {
     case 6:
       *g_irq_handler_reported_error = 1;
       vp_assert_irq(0, 0);
+      *g_special_handler_idx = 0;
+      return;
+      break;
+    case 7:
+    *g_special_handler_idx = 0;
```

Comment:
Could you please indent this part to match the rest of the case-statements?

Reply:
Good catch! Fixed.